src/App.tsx

```diff
@@ -5,10 +5,12 @@ import { Fetcher } from "swr";
 import React from "react";
 import { SenOppfolgingStatusDTO } from "./schema/senOppfolgingStatusSchema";
 import { MikrofrontendLinkPanel } from "./components/panels/MikrofrontendLinkPanel";
+import moment from "moment";
```

Comment:
Kan vurdere å bruke dayjs i stedet, den har mye mindre bundle size

Reply:
Ah, jeg prøver

Reply:
Bytta ut med dayjs nå, men måtte legge på en plugin for at det skulle fungere

`dayjs.extend(customParseFormat);`

Ligger det riktig sted eller skal det ligge et annet sted?

Reply:
Sikkert greit! :) Ikke sikker på hva som er best. Tror jeg har sett det i util-funksjon tidligere.